editor/src/components/navigator/navigator-item/layout-icon.tsx

```diff
@@ -115,7 +115,7 @@ export const LayoutIcon: React.FunctionComponent<React.PropsWithChildren<LayoutI
             category='navigator-element'
             type='zerosized-div'
             testId={iconTestId}
-            color={'main'}
+            color={color === 'subdued' ? 'subdued' : 'main'}
```

Comment:
I think this should also check for the emphasized color too 🤔

Reply:
I went with it not doing it, because it's for a zero sized div, which shouldn't be emphasized I believe.